scripts/delegate-command.mjs

```diff
@@ -0,0 +1,33 @@
+/**
+ * Delegate the provided command to a sub-project.
+ * For example:
+ * `npm run in vue-handsontable build` or `run in vue build`
+ * will run the `build` command from the directory of the provided project.
+ */
+import execa from 'execa';
+
+const [project, command, args] = process.argv.slice(2);
+const PROJECT_ALIASES = {
+  angular: 'angular-handsontable',
+  react: 'react-handsontable',
+  vue: 'vue-handsontable'
+};
+const commandArray = [
+  'run',
+  command
+];
+
+if (args) {
+  commandArray.push(args);
```

Comment:
In conjunction with https://github.com/handsontable/handsontable/pull/7434/files#r570229631.
```suggestion
  commandArray.push(...args);
```